example/package.json

```diff
@@ -12,6 +12,7 @@
   "dependencies": {
     "@react-native-async-storage/async-storage": "^1.17.11",
     "@react-native-community/netinfo": "^9.3.7",
+    "@react-native-picker/picker": "^2.4.9",
```

Comment:
Just want to call it out to verify - did the previous `@react-native-community/picker` require a Pods install as well, or is that new to this newer picker?  No concern either way, just curious

Reply:
Yes, the previous did require it as well.  In fact it is largely the same codebase - just a renamed repo and then continued development.